### ISSUE TITLE
GPG on Mac

### ISSUE THREAD

**Issue by FeLoe** [ccs-amsterdam/compendium#1]
On Mac OS the decryption step only works after manually installing GnuPG, this need to be reflected in the requirements (not sure exactly how at the moment..).

> Unlike most Linux distributions (which make heavy use of GnuPG for ensuring untampered software within their package repositories), Mac OS X does not bring GnuPG with the operating system, so you have to install it on your own.
> 
> Possible sources are:
> 
> Package manager Homebrew: brew install gnupg gnupg2
> Package manager MacPorts: sudo port install gnupg gnupg2
> Install from GPGTools, which also brings GUI applications and integration in Apple Mail
>

**Comment by vanatteveldt** [comment]
Good point. We probably need to think about the "pre-requirements", eg. things that need to be installed before the "doit" can do its work. 

Does the python pip install step work out of the box?
Did you try the R/packrat step?
